src/main/java/adhd/diary/auth/service/GoogleService.java

```diff
@@ -80,21 +80,19 @@ public String getGoogleAccessToken(String code) {
                 JsonNode jsonNode = objectMapper.readTree(response.getBody());
                 return jsonNode.get("access_token").asText();
             } catch (JsonProcessingException e) {
-                logger.error("구글 액세스 토큰을 파싱하는데 실패했습니다.", e);
-                throw new IllegalArgumentException("구글 액세스 토큰을 가져오는데 실패했습니다.", e);
+                throw new LoginNotFoundException(ResponseCode.GOOGLE_TOKEN_RETRIEVAL_FAILED);
             }
         } else {
-            logger.error("구글 액세스 토큰 요청이 실패했습니다. 상태 코드: {}", response.getStatusCode());
-            throw new IllegalArgumentException("구글 액세스 토큰을 가져오는데 실패했습니다.");
+            throw new LoginNotFoundException(ResponseCode.GOOGLE_TOKEN_RETRIEVAL_FAILED);
         }
     }
 
     public OAuth2UserInfo getUserGoogleInfo(String accessToken) {
         HttpHeaders headers = new HttpHeaders();
         headers.add("Authorization", "Bearer " + accessToken);
+        headers.add("Content-type", "application/x-www-form-urlencoded;charset=utf-8");
```

Comment:
```suggestion
        headers.setBearerAuth(accessToken);
        headers.setContentType(MediaType.APPLICATION_JSON);
```
위와 같이 바꿔도 같은 표시로 된다고 하네요!